package.json

```diff
@@ -102,7 +102,7 @@
     "nats.ws": "^1.7.1",
     "qs": "^6.9.4",
     "set-cookie-parser": "^2.5.1",
-    "siwe": "^1.1.6",
+    "siwe": "^2.1.4",
```

Comment:
Question for you @JGiter . As this is a major version upgrade, are there any breaking changes here? If so, do we need to update `siwe` in passport-did-auth/did-auth-proxy in parallel so that the SIWE messages created by iam-client-lib are accepted by the backend apps? Or will messages created with v2 siwe be accepted by servers running v1 siwe?

Reply:
I didn't notice breaking changes. `siwe` v2 already used in ssi-hub and [passport-did-auth](https://github.com/energywebfoundation/passport-did-auth/blob/7bc46205872e090d96fe0d7254f359bc81365726/package.json#L68). I also tested ICL with `siwe` 2.1.3 locally and everything went well. Though there is something happened in 2.1.4, so I am downgrading siwe to 2.1.3

Reply:
Actually problem with 2.1.4 was for some temporal reason. So I'll bring 2.1.4 back